pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/IngestionConfig.java

```diff
@@ -51,6 +51,10 @@ public class IngestionConfig extends BaseJsonConfig {
   @JsonPropertyDescription("Configs related to skip any row which has error and continue during ingestion")
   private boolean _continueOnError;
 
+  @JsonPropertyDescription("If set to true, the records with GenericRow.INCOMPLETE_RECORD_KEY will not be consumed."
+      + "This can be helpful if user only wants to see correct data in the table")
```

Comment:
Correct me if I am mistaken: we are not really storing the partially decoded record. When decode failure happens, we only store an empty `GenericRow` with the field `INCOMPLETE_RECORD_KEY`. So, calling this `skipPartialRecords` is confusing. Something like `DECODE_FAILED_KEY` seems more appropriate. 

Can you clarify the description -  "If set to true, the records with GenericRow.INCOMPLETE_RECORD_KEY will not be **consumed**." -> consumed for query ?

Reply:
This applies to more cases then just decoder. e.g. if an error occurs during converting one of the columns data type, then if `continueOnError` is true, we simply replace it with default value. Hence it will be easier for users to filter such records with this flag.

Reply:
Based on internal discussions, this flag is removed from the schema. The approach looks ugly and until pinot support virtual columns with actual data we can keep it as same.